### DISCUSSION TITLE
Government agency banner to be used across multiple sites

### DISCUSSION THREAD

**Discussion by brockfanning** [uswds/uswds#5664] [General]
Hi all,

Discussions have come up in response to M-23-22 about adding a common banner for an agency at the top of each website that agency has. An example would be a common banner at the top of both justice.gov and fbi.gov, since they're both under DOJ. I assume this would go right above (or below) the "An official website of the United States government" component. I was just wondering if this question has come up in USWDS discussions and whether there might be any guidelines on the subject. Thank you!

**Comment by thisisdano** [comment]
Hi Brock, this is a good question, and I don't think it has a clear answer yet. Personally, I would be cautious about stacking banners and about conveying multiple levels of identity with multiple levels of banners. We're looking at a research question for the first quarter of next year that tries to get at "How does banner order affect the effectiveness of the messages?". Say you have a gov banner, an agency banner, and a site alert banner: how does the order affect perception and comprehension?

But I'm a bit concerned about banner fatigue and banner stacking in general, and I am more interested in solutions that establish federal identity and agency/product branding without a lot of redundant strata of real estate.

But I'm also curious to hear other experience and perspectives.
